test/hotspot/jtreg/vmTestbase/nsk/jvmti/SetLocalVariable/setlocal001.java

```diff
@@ -97,6 +101,10 @@ public void meth02(int step) {
             meth02(step - 1);
         } else {
             checkPoint();
+            if (currThread.isVirtual()) {
+                out.println("meth01: skipping results check for virtual thread");
```

Comment:
Should be "meth02". Similar issues below.

Reply:
Thanks. Fixed now.